tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_Weno.cpp

```diff
@@ -293,8 +294,11 @@ void test_weno_tci_2d() noexcept {
   }
   ();
 
-  const std::array<double, 4> means_no_activation = {{-1., 1., -1., 1.}};
-  const std::array<double, 4> means_activation = {{-1., 1., -1., 0.9}};
+  const double eps = 100.0 * std::numeric_limits<double>::epsilon();
```

Comment:
explain the eps on the next few lines

Reply:
I've removed the epsilon; fundamentally, there was no need for the slopes to be so close to the critical slope that triggers the limiter. There is now a larger value, and a new comment.